StudCity.Frontend/src/components/Scroller/index.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 
-const Scroller = ({ children }) => {
+const Scroller = ({ children, resRef = null }) => {
 	return (
-		<div className='h-full w-full overflow-y-auto'>
+		<div ref={resRef} className='h-full w-full overflow-y-auto'>
```

Comment:
What about rename this `resRef`. And I have concerns with this `resRef = null` because if this is null in a simple scroller component, probably we don't need to add this `ref` to the `div`